### ISSUE TITLE
Wrong input parameters for sklearn.neural_network.MLPClassifier

### ISSUE THREAD

**Issue by YuriyKhoma** [OHDSI/PatientLevelPrediction#21]
MLPClassifier prototype from mlp.py is outdated. Following error occurs

mlp = MLPClassifier(algorithm ='adam',activation='logistic', alpha=alpha, learning_rate='adaptive', hidden_layer_sizes=(size, 2), random_state=seed)

TypeError
: 
__init__() got an unexpected keyword argument 'algorithm'

**Comment by jreps** [comment]
Thanks for letting me know.  It seems the sklearn version I have wants to use algorithm but the latest sklearn version wants solver. I've now removed this input as 'adam' is the default setting so we don't need to specify it.  This error should be fixed now.
